### PR TITLE
chapter event list is now fetched from hub.

### LINKED ISSUE #542: Chapter's events shouldn't use developers.google.com
Instead, it should use our hub.

### PR DESCRIPTION
**Pull request by tasomaniac**
Previously, i don't know why, we were using some hidden api from developers.google.com. I have been thinking to migrate it to hub. I finally did it.

What was the diff between `Event` and `TaggedEvent` classes? It's just that the former is coming from developers.google.com, the latter is from hub. Which does not make sense at all. Removed `TaggedEvent` class now. We only have `Event` class.

Fixes #542

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/api/GroupDirectory.java [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
-public interface GroupDirectory {
-
-    @GET("groups/pulse_stats/")
-    Call<Pulse> getPulse();
-
-    @GET("groups/pulse_stats/{country}/")
-    Call<Pulse> getCountryPulse(@Path("country") String country);
-
-    @GET("events/feed/json")
-    Call<ArrayList<Event>> getChapterEventList(@Query("start") final int start,
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/api/GroupDirectory.java]: Only this endpoint is removed here.

**Comment by friedger** [comment]
TaggedEvents are events that are tagged :-) 
There could be events that are not associated with a GDG in the list of tagged events. At least that is what I remember, like study jam or IO extended. For study jam, they have now removed this option.

**Comment by tasomaniac** [comment]
They are just events at the end. If they are associated just their chapter
id will be null.

**Comment by Splaktar** [comment]
:+1:

**Pull request merged**